Documentation/MixedRealityConfigurationGuide.md

```diff
@@ -306,17 +306,31 @@ If your controller representation in the scene needs to be offset from the physi
 
 <img src="../Documentation/Images/MixedRealityToolkitConfigurationProfileScreens/MRTK_ControllerVisualizationProfile.png" width="650px" style="display:block;">
 
+<a name="editor-utilities"></a>
```

Comment:
Do we need these? I thought markdown auto generated anchors from headers

Reply:
It’s the original format of the file